modules/tinymce/src/plugins/wordcount/main/ts/core/Count.ts

```diff
@@ -14,11 +14,14 @@ import { getText } from './GetText';
 
 export type Counter = (node: Node, schema: Schema) => number;
 
+const removeZwsp = (text: string) =>
+  text.replace('\u200B', '');
```

Comment:
This isn't going to work properly... this will only remove the first instance of a ZWSP character. It needs to be something like this instead:
```suggestion
  text.replace(/\u200B/g, '');
```
Can we please make sure to include multiple characters in one of the tests?

Reply:
Whoops